odoo/addons/base/tests/test_image.py

```diff
@@ -91,6 +95,13 @@ def test_02_image_fix_orientation(self):
         self._orientation_test(7, (pink, yellow, green, blue), size, expected)  # right/bottom
         self._orientation_test(8, (green, blue, pink, yellow), size, expected)  # left/bottom
 
+    def test_03_image_fix_orientation_exif(self):
+        """Test that a png image with exif orientation tag gets rotated"""
+        image = tools.base64_to_image(self.base64_exif_png)
+        self.assertEqual(image.size, (3,6))
+        image = tools.image_fix_orientation(image)
+        self.assertEqual(image.size, (6,3))
```

Comment:
This is failing on runbot.  Which version of Pillow is it running?

Just as a hint because it's too far for me to remember exactly, but it might be possible that some file format had transpose built in them, and this might have been the case of PNG. This could be another possible reason to explain why limiting the transpose to JPEG as well.

I remember very specifically that it was extremely annoying to code `test_02_image_fix_orientation` due to JPEG compression (couldn't compare exact color), and yet I felt I had no choice but to use JPEG to test the transposition custom code.

Reply:
Reply, you a re right, runbot cirrent Docker image  has pillow 5.4.1